lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -166,16 +166,16 @@ def build_request
         xml = Builder::XmlMarkup.new :indent => 2
         xml.instruct! :xml, :encoding => 'UTF-8'
         xml.declare! :DOCTYPE, :paymentService, :PUBLIC, '-//WorldPay//DTD WorldPay PaymentService v1//EN', 'http://dtd.worldpay.com/paymentService_v1.dtd'
-        xml.tag! 'paymentService', 'version' => '1.4', 'merchantCode' => @options[:login] do
+        xml.paymentService 'version' => '1.4', 'merchantCode' => @options[:login] do
```

Comment:
It turns out that these hashes don't need to be enclosed by parentheses after all, so I removed them anywhere they weren't needed.